crates/wasmtime/src/runtime/vm/interpreter.rs

```diff
@@ -82,6 +82,7 @@ impl InterpreterRef<'_> {
     /// The `bytecode` pointer should previously have been produced by Cranelift
     /// and `callee` / `caller` / `args_and_results` are normal array-call
     /// arguments being passed around.
+    #[inline(never)]
```

Comment:
Mind leaving a coment on this as to why?

Reply:
This one was just an intuition that we didn't want to inline anything related to the interpreter. But it also wasn't getting inlined in the first place, afaik, so I'll just remove it.